commands/user-info.js

```diff
@@ -36,9 +37,9 @@ async function getDetails(message) {
 	catch (error) {
 		console.log(error);
 	};
-	console.log(`Finding match for ${message.author.tag}`);
+	console.log(logThemes.info(`Finding match for ${message.author.tag}...`));
```

Comment:
Can you make the logging with this syntax`console.log.info('msg string')`?
It makes easier to code.